src/resonanceReconstruction/rmatrix/src/makeQuantumNumbers.hpp

```diff
@@ -0,0 +1,21 @@
+std::vector< ChannelQuantumNumbers >
+makeQuantumNumbers( const ParticlePair& incident, unsigned int lmax ) {
+
+  std::vector< ChannelQuantumNumbers > numbers;
+  auto channelspins = possibleChannelSpinValues( incident.particle().spin(),
+                                                 incident.residual().spin() );
+  for ( unsigned int l = 0; l < lmax; ++l ) {
```

Comment:
I'm a little surprised you didn't do this with `ranges`. Granted, it is probably more readable this way.